server/src/internalClusterTest/java/org/elasticsearch/cluster/PrevalidateShardPathIT.java

```diff
@@ -39,6 +40,10 @@
 @ESIntegTestCase.ClusterScope(scope = ESIntegTestCase.Scope.TEST, numDataNodes = 0)
 public class PrevalidateShardPathIT extends ESIntegTestCase {
 
+    @TestLogging(
+        value = "org.elasticsearch.cluster.service.MasterService:DEBUG",
+        reason = "https://github.com/elastic/elasticsearch-serverless/pull/2068"
```

Comment:
Should it link https://github.com/elastic/elasticsearch/issues/104807 instead?

Reply:
Thank you for catching this!